script/restore.sh

```diff
@@ -1,29 +1,47 @@
-#!/bin/bash -e
-BDIR=$(readlink -f "$1")
```

Comment:
Tested this and the first thing that came to mind is: we need a confirm on this, similar to katello-remove.  This script is very destructive if you run it accidentally so having a confirmation block like in:

https://github.com/Katello/katello/blob/master/deploy/script/katello-remove#L3-L37

would be very helpful.